### PR TITLE
Additions made to Analyses User Guide

### PR DESCRIPTION
**Pull request by mdhaul**
I've added info about pathoscope analysis. Also added sections on how to:

- analyze a good quality sample
- analyze a bad quality sample
- analyze a contaminated sample
- analyze a healthy sample

Let me know if there is more to add or any changes to make!

### CODE REVIEW

**Comment by mdhaul** [comment]
- I've changed the word "statistics" to value
- made a table for all the values
- copied info that was under the heading "What are analyses" and pasted it under the "Overview" section at the top because it was empty and thought it made more sense at the top?

File: content/docs/manual/ug_analyses/index.md [hunk 1]
```diff
@@ -87,4 +82,72 @@ Navigate to the analysis listing for the sample whose analysis you want to delet
...
+
+When we filter our results we see that all other viruses have low weight, depth, and coverage, assuring us that these are not present in the sample. With these results in mind, we can confidently say that this sample is from a healthy plant.
+
+## Low Quality
+
+![Poor quality analysis](poor_quality_analysis.png)
+
+In this example, only 0.04% of reads are mapped to the total number of reads therefore it is highly unlikely that the pathogens listed are actually present in the sample. Although weight and coverage are high, no sample has a high depth value. 
+
+Additionally when we take a look at the isolate coverage charts we see that the number of reads being mapped are a maximum of 14 which is very small.
```
Comment (igboyes) [inline content/docs/manual/ug_analyses/index.md]: This is not accurate. `14` is the maximum observed depth.

File: content/docs/manual/ug_analyses/index.md [hunk 2]
```diff
@@ -87,4 +82,72 @@ Navigate to the analysis listing for the sample whose analysis you want to delet
...
+![Coverage Charts](isolate_charts3.png)
+
+## Analysis of Contaminated Sample
```
Comment (igboyes) [inline content/docs/manual/ug_analyses/index.md]: I think the **Low Quality** and **Analysis of Contaminated Sample** should be put together. They are all contamination as far as I'm concerned.

File: content/docs/manual/ug_analyses/index.md (hunk 1 above)
Comment (mdhaul) [reply content/docs/manual/ug_analyses/index.md]: I have rephrased it!

**Comment by igboyes** [comment]
You didn't resolve the second issue:
>I think the Low Quality and Analysis of Contaminated Sample should be put together. They are all contamination as far as I'm concerned.

**Comment by mdhaul** [comment]
I thought you wanted the "low quality" and "analysis of contaminated sample" sections put together one after another. Do you want both of these examples under one subheading (ex. just "Low Quality")?

**Comment by mdhaul** [comment]
I've added the "Interpret Nuvs" section to Analysis user guide. Take a look and let me know about any changes need to be made!

**Pull request merged**